libs/dh/market-participant/actors/data-access-api/src/lib/dh-market-party-credentials.store.ts

```diff
@@ -149,7 +149,7 @@ export class DhMarketPartyCredentialsStore extends ComponentStore<CertificateSta
           this.httpClient.v1MarketParticipantActorRequestClientSecretCredentialsPost(actorId).pipe(
             tapResponse(
               (clientSecret) => {
-                this.patchState({ clientSecret });
+                this.patchState({ clientSecret: clientSecret.secretText });
```

Comment:
suggestion: can you also adjust the mock request at "data-access-mocks/src/lib/marketParticipant.ts"

Reply:
Done